update.php

```diff
@@ -1451,14 +1523,17 @@ function update_1163() {
 	$r = q("ALTER TABLE `item` ADD `network` char(32) NOT NULL");
 
 	set_config('system', 'maintenance', 0);
-	if(!$r)
-		return UPDATE_FAILED;
 
-	return UPDATE_SUCCESS;
+	if ($r) {
+		return UPDATE_SUCCESS;
+	}
+
+	return UPDATE_FAILED;
 }
 function update_1164() {
 	set_config('system', 'maintenance', 1);
 
+	/// @TODO If one update fails, should it continue?
```

Comment:
I think, I put further rewrites towards unified database updates in an other branch/PR and not mix with this (coding-convention and unifying updates are two different things).